packages/plugin-ext-vscode/src/browser/plugin-vscode-commands-contribution.ts

```diff
@@ -14,24 +14,23 @@
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
 
-import { injectable, inject } from 'inversify';
-import { CommandContribution, CommandRegistry, Command } from '@theia/core';
+import { Command, CommandContribution, CommandRegistry, ResourceProvider } from '@theia/core';
+import { ApplicationShell, NavigatableWidget, open, OpenerService, Saveable } from '@theia/core/lib/browser';
+import { ContextKeyService } from '@theia/core/lib/browser/context-key-service';
+import { ApplicationShellMouseTracker } from '@theia/core/lib/browser/shell/application-shell-mouse-tracker';
 import { CommandService } from '@theia/core/lib/common/command';
 import TheiaURI from '@theia/core/lib/common/uri';
-import URI from 'vscode-uri';
-import { ContextKeyService } from '@theia/core/lib/browser/context-key-service';
-import { DiffService } from '@theia/workspace/lib/browser/diff-service';
 import { EditorManager } from '@theia/editor/lib/browser';
+import { TextDocumentShowOptions } from '@theia/plugin-ext/lib/common/plugin-api-rpc-model';
+import { DocumentsMainImpl } from '@theia/plugin-ext/lib/main/browser/documents-main';
+import { createUntitledResource } from '@theia/plugin-ext/lib/main/browser/editor/untitled-resource';
 import { WebviewWidget } from '@theia/plugin-ext/lib/main/browser/webview/webview';
-import { ApplicationShell, NavigatableWidget, OpenerService, open, Saveable } from '@theia/core/lib/browser';
-import { ResourceProvider } from '@theia/core';
+import { fromViewColumn, toDocumentSymbol } from '@theia/plugin-ext/lib/plugin/type-converters';
 import { ViewColumn } from '@theia/plugin-ext/lib/plugin/types-impl';
-import { TextDocumentShowOptions } from '@theia/plugin-ext/lib/common/plugin-api-rpc-model';
-import { fromViewColumn } from '@theia/plugin-ext/lib/plugin/type-converters';
 import { WorkspaceCommands } from '@theia/workspace/lib/browser';
-import { createUntitledResource } from '@theia/plugin-ext/lib/main/browser/editor/untitled-resource';
-import { DocumentsMainImpl } from '@theia/plugin-ext/lib/main/browser/documents-main';
-import { ApplicationShellMouseTracker } from '@theia/core/lib/browser/shell/application-shell-mouse-tracker';
+import { DiffService } from '@theia/workspace/lib/browser/diff-service';
+import { inject, injectable } from 'inversify';
+import URI from 'vscode-uri';
```

Comment:
it seems the whole order of imports is changed but it's not defined there https://github.com/eclipse-theia/theia/wiki/Coding-Guidelines so I would untouch the existing order.

Reply:
This is what `organize imports` produces. So if we don't align with that no one can use that.

Reply:
I have added https://github.com/eclipse-theia/theia/wiki/Coding-Guidelines#imports and put it on the dev meetings agenda to raise awareness in case not everyone agrees.